heatpump.yaml

```diff
@@ -2073,52 +2091,157 @@ switch:
   # Register: 0 -> Bit 0
   # When Room Temperature Control is enabled, the temperature sensor in the remote controller
   # will be used to define when the heatpump should be turned (powered) on or off
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Room Temperature Control"
+  - platform: template
+    name: "Room Temperature Control"                  #"Power Climate Control -uses FanCoilUnit as emission type"
     id: "${devicename}_room_temperature_control"
     icon: mdi:eye
-    register_type: holding
-    address: 0x0
-    bitmask: 0x1
+    entity_category: config
+    restore_mode: DISABLED
+    optimistic : true
+    lambda: "return (id(unmasked_value_register_0) & 0x1) == 0x1;"
+    on_turn_on:
+      #- logger.log: "Switch Turned On!"
+      - lambda: |-
+          uint16_t checked_bit = 0x1;
+          uint16_t new_value = id(unmasked_value_register_0);  // The original unmasked value
+          new_value &= ~checked_bit;         //Clear and set
+          new_value += checked_bit;
+          if ((new_value) != id(unmasked_value_register_0)) {
+            ESP_LOGI("unmasked_value_register_0", "Set option to on power_air_conditioner_zone_1 0x%x -> 0x%x", id(unmasked_value_register_0), new_value);
```

Comment:
Can you comment this line. It can stay in, but not enabled by default, so that the log is not overloaded